specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-12-01-preview/FileImports.json

```diff
@@ -226,10 +226,7 @@
             }
           }
         },
-        "x-ms-long-running-operation": true,
-        "x-ms-long-running-operation-options": {
-          "final-state-via": "location"
-        }
+        "x-ms-long-running-operation": true
       }
```

Comment:
whats the reason for these changes?

Reply:
This header was added only because we thought an asynchronous delete was required to have a location header.  We don't use it, and (due to a coding error) no usable value is even being returned (so no other API consumers could be using it).

Recently this header caused the endpoint to be flagged as potentially vulnerable to a security bug, and it seemed the easiest and most prudent thing to do was remove the header altogether.